tests/test-lapp.lua

```diff
@@ -200,3 +209,7 @@ test.asserteq(lapp(spec,{'-vs',200,'-sk',1}),{
   width = 256
 })
 
+
+
+
+
```

Comment:
is this whitespace intentional? can it be removed?

Reply:
whoops added the fix